README.md

```diff
@@ -99,6 +99,9 @@ Minio Server comes with an embedded web based object browser. Point your web bro
 ## Test using Minio Client `mc`
 `mc` provides a modern alternative to UNIX commands like ls, cat, cp, mirror, diff etc. It supports filesystems and Amazon S3 compatible cloud storage services. Follow the Minio Client [Quickstart Guide](https://docs.minio.io/docs/minio-client-quickstart-guide) for further instructions.
 
+## Pre-existing data
+Pre-existing data in the data directory, where minio server stores its objects and buckets, would also be accessible.
```

Comment:
We need to indicate here that the data will only be available in certain modes

Reply:
@nitisht We have not mentioned FS mode anywhere in the document, so I believe we need not specify it.

Reply:
> We have not mentioned FS mode anywhere in the document,

I agree, but since we're talking about data it doesn't hurt to be as clear as possible. Maybe we can say something like this

```
When deployed on a single drive, for example `minio server /mnt/data`. Minio server lets you access pre-existing data in `/mnt/data` directory.
```

This way it is implicit that we're talking about FS mode.